README.md

```diff
@@ -43,6 +43,15 @@ class Thing < ActiveRecord::Base
 end
 ```
 
+You can optionally customize the model's unit accessor column by specifying it in the `unit_column_name` option, as follows:
```

Comment:
I called it `unit_column_name` to be as explicit as possible... Let's see what @kmcphillips thinks. Other options: `unit_field`, `unit_field_name`, `unit_accessor` ...